Assignment2.yaml

```diff
@@ -0,0 +1,415 @@
+  /People/schema:
+    get:
+      tags:
+      - "People"
+      summary: "Return Schema"
+      description: "Returns a json Schema"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/people"
+  /starship:
+    get:
+      tags:
+      - "Starship"
+      summary: "Find list of starships"
+      description: "Returns a List of starship"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "name"
+        in: "query"
+        required: false
+        type: "string"
+      - name: "model"
+        in: "query"
+        required: false
+        type: "string"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/starship"
+  /starship/{starshipId}:
+    get:
+      tags:
+      - "Starship"
+      summary: "Find starship by ID"
+      description: "Returns a single starship"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "starshipId"
+        in: "path"
+        description: "ID of starship to return"
+        required: true
+        type: "number"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/starship"
+  /starship/schema:
+    get:
+      tags:
+      - "Starship"
+      summary: "Find starship by ID"
+      description: "Returns a json schema"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/starship"
+  /species:
+    get:
+      tags:
+      - "Species"
+      summary: "Find list of species"
+      description: "get all the species resources"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "name"
+        in: "query"
+        required: false
+        type: "string"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/species"
+  /species/{speciesId}:
+    get:
+      tags:
+      - "Species"
+      summary: "Find speciese by ID"
+      description: "get a specific species resource"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "speciesId"
+        in: "path"
+        description: "ID of species to return"
+        required: true
+        type: "number"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/species"
+  /species/schema:
+    get:
+      tags:
+      - "Species"
+      summary: "Return Schema"
+      description: "Returns a json Schema"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/species"
+definitions:
+  Vehicle:
```

Comment:
Your Definition headings are 'Vehicle' (First capital letter), 'people', 'starship' (singular) and 'species' (plural).
Is there any specific pattern you are following. Because as I see there is none.